src/contexts/ConfigurationContext.tsx

```diff
@@ -437,8 +440,8 @@ export const ConfigContextProvider = ({ children }: Props) => {
       setFilters({
         isEnabled: isFilteringEnabled,
         allSources: allSources,
-        sources,
-        sourceValueToLabelMap,
+        sources: sources,
+        sourceValueToLabelMap: sourceValueToLabelMap,
```

Comment:
These changes are both unnecessary.

Reply:
Yes I know. But argument names were used in the 2 lines above, so I thought it's clearer like this - for consistency.